test/test_autograd.py

```diff
@@ -8377,15 +8483,18 @@ def functional_op(x):
                 self.assertFalse(func_out.requires_grad)
 
     def test_inference_mode_inf_tensor_in_inf_mode_inplace_op(self):
-        with torch.inference_mode():
-            for requires_grad in (True, False):
-                c = torch.ones(1, 2, 3, requires_grad=requires_grad)
+        def run_test(fn):
+            with torch.inference_mode():
```

Comment:
nit: one way to make this diff nicer by avoiding all the extra indentation is to do:
```python
@torch.inference_mode()
def run_test(fn):
```